<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>951</width>
      <height>561</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_image">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="QLabel" name="image_capteur">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>951</width>
        <height>531</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img_ARDUCAM.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_graphique">
     <attribute name="title">
      <string>Graphique</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="bouton_capture">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>570</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bouton_vider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>640</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Vider</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nbrCapture">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>620</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_nbrCapture">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>620</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre d'it√©ration</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_nbrCapture">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>650</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>561</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuEnregistrer">
     <property name="title">
      <string>Enregistrer</string>
     </property>
     <addaction name="actionGroupe_de_pixels"/>
     <addaction name="actionMoyenne_des_colonnes"/>
    </widget>
    <addaction name="menuEnregistrer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Port</string>
    </property>
    <widget class="QMenu" name="menuNom">
     <property name="title">
      <string>Nom</string>
     </property>
    </widget>
    <addaction name="menuNom"/>
    <addaction name="separator"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menuCommandes">
    <property name="title">
     <string>Commandes</string>
    </property>
    <widget class="QMenu" name="menuDimensions">
     <property name="title">
      <string>Dimensions</string>
     </property>
     <addaction name="action320x240"/>
     <addaction name="action640x480"/>
     <addaction name="action1024x768"/>
     <addaction name="action1280x960"/>
     <addaction name="action1600x1200"/>
     <addaction name="action2048x1536"/>
     <addaction name="action2592x1944"/>
    </widget>
    <widget class="QMenu" name="menuFormat">
     <property name="title">
      <string>Format</string>
     </property>
     <addaction name="actionJPEG"/>
     <addaction name="actionRAW"/>
    </widget>
    <widget class="QMenu" name="menuCapture">
     <property name="title">
      <string>Capture</string>
     </property>
     <addaction name="actionCapture_2"/>
     <addaction name="actionTest"/>
    </widget>
    <addaction name="menuDimensions"/>
    <addaction name="menuCapture"/>
    <addaction name="menuFormat"/>
    <addaction name="separator"/>
    <addaction name="actionSaturation"/>
    <addaction name="actionBrightness"/>
    <addaction name="actionContrast"/>
    <addaction name="actionSharpness"/>
    <addaction name="actionExposure"/>
   </widget>
   <widget class="QMenu" name="menuGraphique">
    <property name="title">
     <string>Graphique</string>
    </property>
    <addaction name="actionMoyenne_colonne"/>
    <addaction name="actionListe_pixels"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPort"/>
   <addaction name="menuCommandes"/>
   <addaction name="menuGraphique"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionGroupe_de_pixels">
   <property name="text">
    <string>Groupe de pixels</string>
   </property>
  </action>
  <action name="actionMoyenne_des_colonnes">
   <property name="text">
    <string>Moyenne colonnes</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="action320x240">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>320x240</string>
   </property>
  </action>
  <action name="action640x480">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>640x480</string>
   </property>
  </action>
  <action name="action1024x768">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1024x768</string>
   </property>
  </action>
  <action name="action1280x960">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1280x960</string>
   </property>
  </action>
  <action name="action1600x1200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1600x1200</string>
   </property>
  </action>
  <action name="action2048x1536">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2048x1536</string>
   </property>
  </action>
  <action name="action2592x1944">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2592x1944</string>
   </property>
  </action>
  <action name="actionJPEG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>JPEG</string>
   </property>
  </action>
  <action name="actionRAW">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RAW</string>
   </property>
  </action>
  <action name="actionSaturation">
   <property name="text">
    <string>Saturation</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionSharpness">
   <property name="text">
    <string>Sharpness</string>
   </property>
  </action>
  <action name="actionExposure">
   <property name="text">
    <string>Exposure</string>
   </property>
  </action>
  <action name="actionMoyenne_colonne">
   <property name="text">
    <string>Moyenne colonnes</string>
   </property>
  </action>
  <action name="actionListe_pixels">
   <property name="text">
    <string>Liste pixels</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionCapture_2">
   <property name="text">
    <string>Capture</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
